Makefile

```diff
@@ -396,6 +396,10 @@ test: $(LINUXKIT) test-images-patches | $(DIST)
 	$(QUIET)$(DOCKER_GO) "cd \"$(GOTREE)\"; ../../tools/fuzz_test.sh" $(GOTREE) $(GOMODULE)
 	$(QUIET): $@: Succeeded
 
+test-commits:
+	GIT_SEQUENCE_EDITOR=/bin/true git rebase -i --exec 'make test' origin/master
```

Comment:
wow, this is a nice git command! didn't know something like this was possible :+1: